omnibus/files/server-ctl-cookbooks/infra-server/libraries/private_chef.rb

```diff
@@ -309,8 +313,8 @@ def gen_frontend
       PrivateChef[:role] = 'frontend'
       PrivateChef['bookshelf']['enable'] ||= false
       PrivateChef['bookshelf']['vip'] ||= PrivateChef['backend_vips']['ipaddress']
-      PrivateChef['redis_lb']['enable'] ||= false
```

Comment:
I think we'll want to keep redis_lb.enable , and change this to '= false' instead of ||=false